sacred/experiment.py

```diff
@@ -260,17 +261,27 @@ def run_commandline(self, argv=None):
 
         try:
             return self.run(cmd_name, config_updates, named_configs, {}, args)
-        except Exception:
-            if not self.current_run or self.current_run.debug:
-                raise
+        except Exception as e:
+            if not self.current_run or not self.current_run.debug \
+                    or not self.current_run.pdb:
```

Comment:
Confusing if-statement. Are you sure this is what you want? The only case in which this will not execute is if there is a current_run *and* debug is true *and* pdb is true,